### PR TITLE
Iow 421

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Send artifacts to chs artifactory rather than eros artifactory or gitlab

Description
-----------
JIRA: https://internal.cida.usgs.gov/jira/browse/IOW-421

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: Jenkins.build [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@Library(value='iow-ecs-pipeline@1.0.0', changelog=false) _
+@Library(value='iow-ecs-pipeline@single-registry', changelog=false) _
```
Comment (ssoper-usgs) [inline Jenkins.build]: I should have made this a proper draft PR.  These branch/tag references will change to something like 1.1.0 when changes from IOW-429 go live.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). (Patch version X.Y.0 is implied if not specified.)
 
 ## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/0.4.0...master)
+### Changed
```
Comment (ayan-usgs) [inline CHANGELOG.md]: I'm not sure we need a changelog entry representing our internal operations.

File: pipeline.yml [hunk 3]
```diff
@@ -19,4 +18,9 @@ healthCheck: "/api/observations/about/health"
...
 contextPath: "/api/observations/*"
 configRepoPath: "wma/docker/wdfn/time-series-services"
-configCredentialsId: "wma-eto-eb-rsa"
\ No newline at end of file
+configCredentialsId: "wma-eto-eb-rsa"
+urlRules:
+  - type: forward
+    conditions:
+      - "/api/observations/*"
+    priority: 302
```
Comment (ayan-usgs) [inline pipeline.yml]: This should be `309` (i.e. the same value as the `rulePriority` parameter). Additionall, the `contextPath` and `rulePriorty` parameters don't do anything anymore and can be removed.

File: CHANGELOG.md (hunk 2 above)
Comment (dsteinich) [reply CHANGELOG.md]: I concur, these will get out-of-hand if we list more than changes visible to the consumer of the api.

**Pull request merged**